woocommerce/woocommerce-gutenberg-products-block
# Update build process

This PR updates the build process to use webpack/babel/etc as used by gutenberg and wc-admin. It removes the entire Grunt workflow, as CSS is built via webpack now. We also remove the build files from the repo. These will still be included when we publish the plugin, but now developers (and anyone working from github) will need to build the project before using it.

In a following PR I'll be adding eslint, and future PRs will add a test framework and CI script, perhaps renovate to keep packages up-to-date.

**How to test these changes:**

1. Look at wp-admin, there should be a message warning you to build the project.
2. Run `npm install` to update your local packages, and `npm build` to build the files
3. Expect: No more message in wp-admin
4. Try to edit a page with a product block, or add a new product block
5. Expect: No functionality change
